processing/bq/test/fixtures/in.js

```diff
@@ -0,0 +1,245 @@
+module.exports = {
```

Comment:
By the way can you rename the fixtures folder before submitting? Maybe something like `voronoi_fixtures`. I'm worried it could get messy when we add more functions.

Reply:
No problem!

Reply:
Thx!